### ISSUE TITLE
[FR]: Generate the package into the Assets/Fonts class

### ISSUE THREAD

**Issue by AlexV525** [FlutterGen/flutter_gen#399]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the problem

Let's say I have a widget that accepts asset paths for displays. In order to make it feasible, I provided `String? icon` and `String? iconPackage` as parameters. Now, if one of my packages uses gen, and generated `UIAssets`, the current usage would be:
```dart
return CustomWidget(
  icon: UIAssets.icons.someIcon.path,
  iconPackage: 'package_name',
);
```

The `package_name` is hard-coded, which means it cannot be referred to when I'm trying to use another icon from the same package.

### Describe the solution

Generate `package` for the class. For example:

```dart
class UIAssets {
  UIAssets._();

  static const String package = 'ui_package'; // <-- Adding this.

  static const $AssetsIconsGen icons = $AssetsIconsGen();
}
```

### Additional context

For the font class, we can apply a similar implementation. However, #357 should be addressed first.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Sevastyan** [comment]
Same for `.rive` generated code.

**Comment by AlexV525** [comment]
@Sevastyan The rive generation should be included in the assets category, which means is implemented by #401.
